src/Taboo/TabooPlayer.py

```diff
@@ -4,7 +4,7 @@
 
 class TabooPlayer:
 
-    def __init__(self, txQueue, name, team):
+    def __init__(self, txQueue, name, team, roundsPlayed=0):
```

Comment:
Maybe does not need to passed in Ctor. Always init to 0.

Reply:
When we allow restarting the server and loading state from mysql/local file/whatever other storage, we'll need the ability to supply these values which is why it helps to make this configurable.